README.md

```diff
@@ -148,6 +148,7 @@ _Software Solution Support Levels_
 | Software<br>Solution | Command<br>Type | Basic<br>Control | Python API<br>Validation<br>Status |
 | -------------------- | --------------- | ---------------- | ---------------------------------- |
 | DPOJET               | PI              | ✅                | ✅                                  |
+| SourceXpress         | PI              | ✅                | ✅                                  |
```

Comment:
Add an entry to the glossary explaining what this Software Solution is.

Reply:
You will also need to update the `macros.py` file to properly update the readme when the docs are generated.